packages/react-scripts/config/webpack.config.js

```diff
@@ -427,9 +433,14 @@ module.exports = function (webpackEnv) {
                   loader: require.resolve('@svgr/webpack'),
                   options: {
                     prettier: false,
-                    svgo: false,
+                    svgo: true,
```

Comment:
Not blocking but we could consider not even needing this option as the latest version of `svgr/webpack` sets this to true by default: https://react-svgr.com/docs/options/#svgo